### PR TITLE
KAFKA-10787: Apply spotless to `stream:test-utils` and `:streams:upgrade-system-tests-xxxx` 

### PR DESCRIPTION
**Pull request by gongxuanzhang**
This PR is sub PR from https://github.com/apache/kafka/pull/16097.
It is part of a series of changes to progressively apply [spotless plugin(import-order)] across all modules. In this step, the plugin is activated in the 
* stream:test-utils
* streams:upgrade-system-tests-0100

## Module and history 

> Please see the table below for the historical changes related to applying the Spotless plugin

| module        | apply |  related PR  |
| -------------- | ------------------- | ------------------- |
| :clients | ❌    | future |
| :connect:api | ✅    | https://github.com/apache/kafka/pull/16299 |
| :connect:basic-auth-extension | ✅    | https://github.com/apache/kafka/pull/16299 |
| :connect:file | ✅    | https://github.com/apache/kafka/pull/16299 |
| :connect:json | ✅   | https://github.com/apache/kafka/pull/16299 |
| :connect:mirror | ✅   | https://github.com/apache/kafka/pull/16299 |
| :connect:mirror-client | ✅   | https://github.com/apache/kafka/pull/16299 |
| :connect:runtime | ❌    | future |
| :connect:test-plugins | ✅   | https://github.com/apache/kafka/pull/16299 |
| :connect:transforms | ✅   | https://github.com/apache/kafka/pull/16299 |
| :core | ❌    | future |
| :examples |  ✅    | https://github.com/apache/kafka/pull/16296 |
| :generator |  ✅     | https://github.com/apache/kafka/pull/16296 |
| :group-coordinator:group-coordinator-api | ✅   | https://github.com/apache/kafka/pull/16298 |
| :group-coordinator | ✅  | https://github.com/apache/kafka/pull/16298 |
| :jmh-benchmarks |  ✅     | https://github.com/apache/kafka/pull/16296 |
| :log4j-appender |  ✅     | https://github.com/apache/kafka/pull/16296 |
| :metadata | ✅     | https://github.com/apache/kafka/pull/16297 |
| :server | ✅     | https://github.com/apache/kafka/pull/16297 |
| :shell |  ✅     | https://github.com/apache/kafka/pull/16296 |
| :storage | ✅     | https://github.com/apache/kafka/pull/16297 |
| :storage:storage-api | ✅     | https://github.com/apache/kafka/pull/16297 |
| :streams | ❌    | future |
| :streams:examples |  ❌   | future |
| :streams:streams-scala | ❌    | future |
| :streams:test-utils |  ✅     | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-0100 |  ✅     | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-0101 | ✅    | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-0102 | ✅    | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-0110 | ✅    | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-10 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-11 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-20 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-21 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-22 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-23 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-24 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-25 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-26 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-27 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-28 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-30 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-31 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-32 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-33 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-34 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-35 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-36 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :streams:upgrade-system-tests-37 | ✅   | https://github.com/apache/kafka/pull/16357 |
| :trogdor |  ✅       | https://github.com/apache/kafka/pull/16296 |
| :raft |  ✅    | https://github.com/apache/kafka/pull/16278 |
| :server-common | ✅    | https://github.com/apache/kafka/pull/16172 |
| :transaction-coordinator | ✅    | https://github.com/apache/kafka/pull/16172 |
| :tools |  ✅   |  https://github.com/apache/kafka/pull/16262  |
| :tools:tools-api | ✅   | https://github.com/apache/kafka/pull/16262 |

## How to test:
We can run `./gradlew spotlessCheck`  check for code that does not meet requirements. 
If we get report that error , we can run `./gradlew spotlessApply` to review my code.
In this PR, all change(exclude `build.gradle`) ` by `spotlessApply`

### CODE REVIEW

File: connect/runtime/src/test/java/org/apache/kafka/connect/runtime/AbstractHerderTest.java [hunk 1]
```diff
@@ -98,7 +99,8 @@
 import static org.mockito.Mockito.when;
 import static org.mockito.Mockito.withSettings;
 
-@RunWith(MockitoJUnitRunner.StrictStubs.class)
+@ExtendWith(MockitoExtension.class)
```
Comment (chia7712) [inline connect/runtime/src/test/java/org/apache/kafka/connect/runtime/AbstractHerderTest.java]: those changes are unrelated to this PR, right?

Comment (gongxuanzhang) [reply connect/runtime/src/test/java/org/apache/kafka/connect/runtime/AbstractHerderTest.java]: o my bad sorry

**Review by chia7712 — APPROVED** [review]
LGTM
